Microsoft.Toolkit.Uwp.SampleApp/SamplePages/StringExtensions/StringExtensionsPage.xaml

```diff
@@ -0,0 +1,28 @@
+﻿<Page
+    x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.StringExtensionsPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp.SamplePages.StringExtensions"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    mc:Ignorable="d">
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+        <StackPanel Orientation="Vertical" VerticalAlignment="Center" HorizontalAlignment="Center">
+            <TextBox PlaceholderText="Enter String" HorizontalAlignment="Center" VerticalAlignment="Center" x:Name="InputTextBox" MinWidth="300" />
+            <TextBlock x:Name="IsValid" Padding="0,5"/>
+        </StackPanel>
+        <controls:WrapPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center" Margin="20" Grid.Row="1">
+            <Button Tapped="Button_Tapped" Content="Is Valid Email?" Margin="10" Tag="0"/>
```

Comment:
You should use the `Shell.Current.RegisterNewCommand` paradigm instead of buttons in the XAML.

Reply:
I tried that but it was not receiving entered text in TextBox. Also there is no XAML in Right hand panel. Just documentation. So i thought instead of spending time in figuring out how this is implemented, i could simply do this.

Reply:
You should have a reference to `InputTextBox` just like any other Page?

```
        protected override void OnNavigatedTo(NavigationEventArgs e)
        {
            Shell.Current.RegisterNewCommand("Is Valid Email?", (s, e2) =>
            {
                IsValid.Text = string.Format("IsValid: {0}", InputTextBox.Text.IsEmail()); 
            });
```

What error was it giving you?

Reply:
Not even an error. It was showing InputTextBox.Text is Empty.

Reply:
My Mistake. Initially i had 5 different TextBoxes for each RegisterNewCommand and i messed up with Names of `TextBox`. Ctrl+Z showed me my stupidity. Changing now.